client/app/components/queries/ApiKeyDialog/index.jsx

```diff
@@ -56,11 +56,11 @@ function ApiKeyDialog({ dialog, ...props }) {
 
         <h5>Example API Calls:</h5>
         <div className="m-b-10">
-          <label>Results in CSV format:</label>
+          <span>Results in CSV format:</span>
```

Comment:
Maybe revert this and address it in a followup? It feels like `label` is appropriate, and that `CodeBlock` should use a `textarea` tag instead of a `code` tag. That way you can preserve the usefulness of the `label` element.